flocx_market/db/sqlalchemy/api.py

```diff
@@ -0,0 +1,36 @@
+from oslo_utils import uuidutils
+
+from flocx_market.db.orm import orm
+from flocx_market.db.sqlalchemy import models
+
+
+def offer_get(marketplace_offer_id):
```

Comment:
I see that Ironic puts all the database methods inside a subclass of  `ironic.db.api`, rather than using module level methods.  What do we think about that?

Reply:
Yep! I saw that, but Nova (and Mistral) do not, so I, uh, went with the simpler direction. I don't see it impacting the usage of the database api to any great degree either way. If you feel strongly about using a subclass though, I'm happy to put it in!